widget_driver_generator/lib/src/model_visitor.dart

```diff
@@ -45,23 +48,36 @@ class ModelVisitor extends SimpleElementVisitor<void> {
 
   @override
   void visitFieldElement(FieldElement element) {
+    if (element.isRedundantToPropertyAccessorElement) {
+      return;
+    }
     final validAnnotationType = _elementUtils.getValidAnnotation(
       element: element,
       validAnnotationTypes: _validAnnotationTypes,
     );
-    if (validAnnotationType != null) {
-      fields.add(AnnotatedElement.fromElement(_elementUtils, element, validAnnotationType));
+    if (!element.name.startsWith('_')) {
```

Comment:
Should we extract this to some helper method?
Like `if (!element.isPrivate) {` or `if (element.isNotPrivate) {`

Reply:
The api already existed, I replaced it 👍